addons/test_discuss_full/tests/test_performance.py

```diff
@@ -108,6 +110,7 @@ def _get_init_messaging_result(self):
             The point of having a separate getter is to allow it to be overriden.
         """
         return {
+            'lastNotifId': 0,
```

Comment:
Instead of hardcoding 0 it would be better to compare against `_bus_last_id` read at the proper time in the test.